### DISCUSSION TITLE
What the args looks like without explicit recombinations

### DISCUSSION THREAD

**Discussion by hyanwong** [tskit-dev/what-is-an-arg-paper#192] [General]
Just for interest, here's what the inference program results look like when we simplify out the recombination nodes (but keep unary in coalescent). This is a reasonable way to compare tsinfer with KwARG and ARGweaver.
<img width="1250" alt="Screenshot 2022-04-05 at 11 39 29" src="https://user-images.githubusercontent.com/4699014/161736582-ee035fee-26d9-458b-b562-121580c38b38.png">

**Comment by hyanwong** [comment]
This also emphasises that `tsinfer` has too many long edges, presumably because if there is an equal likelihood of matching against a number of different ancestors, we always pick the oldest.

It would be interesting to see if this changes as we change the mismatch rate.

**Comment by hyanwong** [comment]
And here's what full simplification looks like, which is a reasonable way to compare relate with all the others. Note how similar the edge diagrams look in the first 3.
<img width="1256" alt="Screenshot 2022-04-05 at 11 40 22" src="https://user-images.githubusercontent.com/4699014/161736746-330fe323-424d-4632-81e3-e5d5a207d1a8.png">

**Reply by hyanwong** [reply]
I've just tidied up the point positioning, so it uses the same positions as in the original Fig 6. The KwARG and ARGweaver examples in this full simplified version are pretty revealing, I think. I would consider including something like this in an appendix.

**Reply by jeromekelleher** [reply]
Sure, or supplemental

**Reply by hyanwong** [reply]
Yep, I think I meant supplemental, but my fingers typed "appendix"!
